### ISSUE TITLE
CPU is going crazy

### ISSUE THREAD

**Issue by vcvrackideas** [gle-bellier/Diapason-modules#5]
Hi I've got 87% cpu on my xeon 2620v3. Hope it could be fixable)

**Comment by lebellig** [comment]
Hi, 
Indeed this module have some CPU usage issues. I know some tricks and have some clues to improve this aspect before the first release ;) Thanks for your comment.

**Comment by pgatt** [comment]
There are some tips here if you need them:

Coming from Bruce Frazer who develops Squinky Labs.

[VCO optimisation](https://community.vcvrack.com/t/how-to-make-or-analyze-a-vco-shameless-plug/10713)

**Comment by lebellig** [comment]
Yes it's my main source of inspiration for new improvements ! Such a great document
